amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -211,9 +213,10 @@ class TableDetail extends React.Component<TableDetailProps & RouteComponentProps
                   </EditableSection>
                 </section>
               </section>
-              {data.programmatic_descriptions.length > 0 &&
+              {
+                data.programmatic_descriptions.length > 0 &&
```

Comment:
@ttannis @feng-tao same issue with the truthy conditions here as well. if this value is null then this is breaking the UI. 😢

Reply:
@verdan is this for the case of the quickstart as well? In practice the metadata service is supposed to be returning an empty array/list for this. 

We can fix all these cases but I do want to understand if this is either a confirmed quickstart issue, if it is affecting your use of Amundsen at ING, or if you're just calling this out to be addressed in the future. Will help prioritize when I go through and look for all these cases that may have been introduced in 2.1.0

Reply:
@ttannis 
I am using the latest version of both frontend and metadata and it's failing. 

Two things here: 
1. This is only implemented on the Neo4j proxy in metadata, hence not available in atlas proxy for now, and returning default value. 
2. `Table` model in amundsen common says the default value of this variable is None. i.e., `programmatic_descriptions: Optional[List[ProgrammaticDescription]] = None`

However, in my opinion it should be 
`programmatic_descriptions: Optional[List[ProgrammaticDescription]] = []` 

or even this will work: 
`programmatic_descriptions: List[ProgrammaticDescription] = []`

Reply:
let me know if I can be of any help. It seems like the amundsen common change would be an easy solution?

Reply:
@samshuster I think that would be correct. I was under the impression the metadata service would be returning an empty list, and should have sanity checked the code in the metada library to ensure this was the case.

@verdan thanks for flagging this. Either of you can feel free to fix if you are able to get to it before I can, and we will be sure to get the PR through and create a new release to unblock as soon as possible.